pearcej/cppds
# Add "Tree Components" to index

### Description
This PR makes the different components of the tree "terms" and adds them to the index.

### Related Issue
fix #332 

### Testing
1. Check out this branch and build locally.
2. Navigate to the index section and make sure the tree terminology appears.

![image](https://github.com/pearcej/cppds/assets/97637517/451bf0c0-14b3-4277-a1ad-4c7a006abd5e)
